test/test_data/firefox.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2022 Orange
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+mapping_test_data = [
+    {
+        "product": "firefox",
+        "version": "105.0.1",
+        "version_strings": ["firefox/105.0.1"],
+    },
+    {
+        "product": "firefox",
+        "version": "106.0.3",
+        "version_strings": ["firefox/106.0.3"],
+    },
+]
+package_test_data = [
+    {
+        "url": "https://kojipkgs.fedoraproject.org/packages/firefox/106.0.3/1.fc38/x86_64/",
+        "package_name": "firefox-106.0.3-1.fc38.x86_64.rpm",
+        "product": "firefox",
+        "version": "106.0.3",
+    },
+    {
+        "url": "http://ftp.fr.debian.org/debian/pool/main/f/firefox/",
+        "package_name": "firefox-en-gb-105.0.1-1_all.deb",
```

Comment:
This package doesn't exist in the above `url`. You can check by visiting the url and see if any package exists with this name.